src/System.Net.Http/tests/FunctionalTests/ResponseStreamTest.cs

```diff
@@ -90,6 +91,54 @@ await client.GetAsync(HttpTestServers.RemoteEchoServer, HttpCompletionOption.Res
             }
         }
 
+        [ActiveIssue(6231, PlatformID.Windows)]
+        [Fact]
+        public async Task ReadAsStreamAsync_InvalidContentLengthResponse_ThrowsIOException()
```

Comment:
I would change "InvalidContentLengthResponse" to "IncompleteContentLengthResponse"